test/rules/no-restricted-globals.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	valid: [
+		"var regex1 = /\\w+/;"
```

Comment:
I just noticed: Jessie omits "var" so all code that is accidentally using "var", both success and failure, should be changed to "const" or "let" depending. We also need a failure test for "var"

Reply:
I do. If a test case has multiple violations, then your checker could legitimately complain about any of them. Unless you're checking something to do with multiple violations, all other test cases should only have the single violation you're testing.